bugwarrior/services/jira.py

```diff
@@ -347,10 +349,13 @@ def get_service_metadata(self):
 
     @classmethod
     def validate_config(cls, service_config, target):
-        for option in ('username', 'password', 'base_uri'):
+        for option in ('username', 'base_uri'):
             if option not in service_config:
                 die("[%s] has no 'jira.%s'" % (target, option))
 
+        if 'password' in service_config and 'PAT' in service_config:
```

Comment:
An exception should be raised if neither option is set.

Reply:
Thanks, updated.

Reply:
I'm not sure your updated logic works the way you think it does. It will catch if both are empty or if both are set to the same string, but not if both are set to different strings. I don't think there's a way to do what you want with a single logical operator. You could do:

```py
if ((not 'password' in service_config or 'PAT' in service_config) or
    ('password' in service_config and 'PAT' in service_config)):
```

Reply:
It does also catch the case where they are set to different strings... each `('...' in service_config)` evaluates as a Boolean value, so the full condition is: 'are both Boolean values the same?' i.e. both present or both absent.

I think your suggestion would trigger when 'PAT' is set but 'password' is not.

Is this more readable?:
```
password_set = 'password' in service_config
pat-set = 'pat' in service_config
if (password_set and pat_set) or not (password_set or pat_set):
```

Reply:
Ah, yeah the `not` may need to be outside the second parenthesis. Use whichever route you find the most readable.

Reply:
Updated.